internal/cluster/cluster.go

```diff
@@ -123,6 +123,7 @@ type API interface {
 	DeregisterInactiveCluster(ctx context.Context, maxDeregisterPerInterval int, inactiveSince strfmt.DateTime) error
 	TransformClusterToDay2(ctx context.Context, cluster *common.Cluster, db *gorm.DB) error
 	RefreshSchedulableMastersForcedTrue(ctx context.Context, clusterID strfmt.UUID) error
+	UploadEvents(ctx context.Context, clusterID *strfmt.UUID)
```

Comment:
why is that part of a cluster api?

Reply:
I might be misunderstanding, but since we're uploading by cluster that it could exist as a cluster API. Is it better to leave it as a function that's not part of the API?